client/signup/steps/plans/style.scss

```diff
@@ -1,6 +1,11 @@
 .plans-step {
 	margin: 0 auto;
 	max-width: 700px;
+
+	&.is-wide-layout {
```

Comment:
So only the redesigned plans need a bigger max-width but the old one will coexist?